airflow/www/views.py

```diff
@@ -2627,6 +2627,9 @@ class TaskInstanceModelView(AirflowModelView):
                     'unixname', 'priority_weight', 'queue', 'queued_dttm', 'try_number',
                     'pool', 'log_url']
 
+    # this is to remove the sorting option on these columns that raise `property has no attribute asc`
+    order_columns = [item for item in list_columns.copy() if item not in ['try_number', 'log_url']]
```

Comment:
I think this can do w/out the comment, since `order_columns` has a specific meaning/purpose in Flask AppBuilder. Otherwise this looks good to me.

Reply:
I agree with you. Have removed the comment. And also removing the .copy(). Thanks for your help on this